test/chord_symbol/to_string.test.ts

```diff
@@ -61,6 +61,28 @@ describe('Chord', () => {
           expect(chord.toString()).toEqual('Eb');
         });
       });
+
+      describe('with option unicodeModifer:true', () => {
+        describe('with sharp modifer', () => {
```

Comment:
This is off, as the first test is about flats. Maybe remove this `describe` scope and change the descriptions to something like `returns the right string representation for flats` and `returns the right string representation for sharps`